estimatey-api/Estimatey.Infrastructure/DependencyInjection.cs

```diff
@@ -19,7 +19,7 @@ public static IServiceCollection AddInfrastructure(this IServiceCollection servi
         services.AddScoped<WorkItemSynchronizationService>();
         services.AddScoped<LinksSynchronizationService>();
 
-        services.AddHostedService<ProjectsSynchronizationService>();
+        //services.AddHostedService<ProjectsSynchronizationService>();
```

Comment:
Did you mean to commit this commented out?

Reply:
Nope, good spot.  I had just disabled it so that I could focus on the logged time syncing in the logs.